rslib/src/decks/schema11.rs

```diff
@@ -383,6 +396,13 @@ impl From<Deck> for DeckCommonSchema11 {
     }
 }
 
+/// See [crate::deckconfig::schema11::clear_other_duplicates()].
+fn clear_other_duplicates(other: &mut HashMap<String, Value>) {
+    for key in ["review_limit", "new_limit"] {
```

Comment:
Needs to be camelCase, due to `#[serde(rename_all = "camelCase")]`

Reply:
Strange, I thought I had checked, but makes sense. Good catch!